andela/ah-baqi-frontend
# 166980215 Like Dislike UI

### What does this PR do?
Implements liking and disliking comments
### Description of Task to be completed?
- [x] Adds the functionality to like and dislike 
- [x] Test the functionality above
### How can this be manually tested?
- Checkout to:

      branch
     
- Install dependencies:

      yarn
     
- Run local server:

      yarn develop
On a specific article, you should be able to like and dislike an article by clicking a heart
### Any background context you want to provide?
None
### What are the relevant pivotal tracker stories?
[#166980231](https://www.pivotaltracker.com/story/show/166980231)
### Screenshots (if appropriate)
